### ISSUE TITLE
Keep newlines for long type annotations

### ISSUE THREAD

**Issue by avh4** [avh4/elm-format#142]
Split from #139 

```elm
notifyServerOfSuccessfulUpload : SlideshowMetadata 
                                 -> AuthToken 
                                 -> Json.Decode.Decoder (Result (RichError c) d) 
                                 -> (Asset a b -> Result (RichError c) d -> EditorAction) 
                                 -> Maybe (Asset a b) 
                                 -> Effects EditorAction
```

**Comment by avh4** [comment]
The formatting should match other multiline type annotations, e.g.

```elm
notifyServerOfSuccessfulUpload :
  SlideshowMetadata
  -> AuthToken 
  -> Json.Decode.Decoder (Result (RichError c) d) 
  -> (Asset a b -> Result (RichError c) d -> EditorAction) 
  -> Maybe (Asset a b) 
  -> Effects EditorAction
```

**Comment by avh4** [comment]
Fixed in 9079a11.
